docs/blog/newsletter-july-2022.rst

```diff
@@ -0,0 +1,118 @@
+
+-------------------------------------
+Toward a theory of technical writing?
+-------------------------------------
+
+A question about whether documentarians suffer from imposter syndrome because we fail to appreciate our own subject matter expertise, and instead tend to defer to other SMEs, sparked a lengthy and thoughtful conversation about whether we lack a formal theory about the discipline.
+
+It was generally agreed that a major consideration is technical writing's focus on the practical, not the theoretical. Software documentation helps its audience get a job done. Its primary purpose is to inform and instruct and while these goals *can* be informed by theories from cognitive science and linguistics, they don't *need* to be. Most documentarians don't have formal training in the work they do. While code still publicly stands -- or at least is perceived to stand -- on the foundations of computer science and its theories, technical communication as an academic discipline tends to be further removed from real-world practices of documentarians.
+
+Ranging further afield, folks pointed to documentation's crucial work in defining audience, creating a single source of truth for the information products need to help their users, and establishing a role in designing and building the products themselves. Perhaps one contributor summarized it best when they said that technical writing "is a diverse field that pragmatically applies theories from other fields as they make sense in the situation." In other words, we look to theory that's informed by practice, rather than the other way around.
+
+Toward the end of the wide-ranging conversation, folks dug into the question of documentation's role in persuading users (in addition to informing and instructing). While some pointed out that this role is conventionally associated with marketing and that marketing language tends to get in the way of the primary goals of informing and instructing, the consensus was that good documentation can and does persuade users to either start with a product or continue with it. Documentation does all the things! ... which might be part of why it's so difficult to provide it with a useful theoretical foundation.
+
+------------------
+What we’re reading
+------------------
+
+The #bipoc group’s been discussing the following materials on diversity, inclusion, and equity. Want to join the conversation? Please join us in the `#bipoc Slack channel <https://writethedocs.slack.com/archives/C016STMEWJD>`_!
```

Comment:
Hm these links are formatted with a single underscore -- maybe the double underscores above are a linting thing?